addon/mixins/route.js

```diff
@@ -160,117 +174,150 @@ export default Ember.Mixin.create({
     delete options.perPage;
     delete options.modelPath;
 
-    this.set('_perPage', perPage);
-    this.set('_modelPath', modelPath);
-    this.set('_extraParams', options);
-
-    var requestPayloadBase = {};
-    requestPayloadBase[this.get('perPageParam')] = perPage;
-    requestPayloadBase[this.get('pageParam')] = startingPage;
+    this.setProperties({
+      currentPage: startingPage,
+      _perPage: perPage,
+      _modelPath: modelPath,
+      _extraParams: options
+    });
 
     if (typeof boundParams === 'object') {
       this.set('_boundParams', boundParams);
-      options = this._includeBoundParams(options, boundParams);
     }
 
-    var params = Ember.merge(requestPayloadBase, options);
-    let promise = this.store[this._storeFindMethod](modelName, params);
+    return this._loadNextPage();
+  },
+
+  /**
+   Trigger a load of the next page of results.
+
+   @method infinityLoad
+   @return {Boolean} always false, to prevent bubbling
+   */
+  _infinityLoad() {
+    if (this.get('_loadingMore')) {
+     return false;
+    }
+
+    if (!this.get('_canLoadMore')) {
+      this._finishLoad();
+      return false;
+    }
+
+    this._loadNextPage();
+
+    return false;
```

Comment:
So I don't think it's actually necessary to `return false;` from this method. The action itself does not return false, or return the value of this method, so it is probably unnecessary.